Mathlib/NumberTheory/PrimeCounting.lean

```diff
@@ -102,7 +107,27 @@ theorem primeCounting'_add_le {a k : ℕ} (h0 : 0 < a) (h1 : a < k) (n : ℕ) :
 theorem zeroth_prime_eq_two : nth Prime 0 = 2 := nth_count prime_two
 
 /-- The `n`th prime is greater or equal to `n + 2`. -/
-lemma add_two_le_nth_prime (n : ℕ) : n + 2 ≤ nth Prime n :=
+theorem add_two_le_nth_prime (n : ℕ) : n + 2 ≤ nth Prime n :=
   zeroth_prime_eq_two ▸ (nth_strictMono infinite_setOf_prime).add_le_nat n 0
 
+theorem surjective_primeCounting' : Function.Surjective π' :=
```

Comment:
Looking at the file, it seems like `primeCounting'_add_le` is the main result. Since these theorems don't depend on it, it might be best to put them above.